File: Jardin De infantes/EE/Servicios/Gestion usuario/sessionManager.vb
Public NotInheritable Class sessionManager

    Private Sub New()

    End Sub
    Private Shared _intance As sessionManager = Nothing

    Public Shared ReadOnly Property intance() As sessionManager
        Get
            If _intance Is Nothing Then
                _intance = New sessionManager
            End If
            Return _intance
        End Get
    End Property

    Private Property usuario As Usuario
    Public Sub login(user As Usuario)
        _usuario = user
    End Sub
    Public Function getUsuario() As Usuario
        Return usuario
    End Function

    Public Sub Logout()
        usuario = Nothing
    End Sub
End Class

File: Jardin De infantes/Seguridad/EEseguridad/Login/SessionManager.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace EEseguridad.Login
{
 public  sealed class SessionManager
    {
        private SessionManager() { }
        private static SessionManager _intance = null;
        public static  SessionManager instance
                    {
           get
            {
                if (_intance == null)
                {
                    _intance = new SessionManager();
                }

                return _intance;
            }
            
        }

        private eeUsuario usuario;
        public void login(eeUsuario user)
        {
            usuario = user;

        }
        public eeUsuario GetUSuario()
        {
            return usuario;
        }
        public void LogOut()
        {
            usuario = null;
        }
    }
}
